src/Adapter/Module/Repository/ModuleRepository.php

```diff
@@ -133,6 +133,11 @@ public function getActiveModulesPaths(): array
         return $this->activeModulesPaths;
     }
 
+    public function clearActiveModulesPaths(): void
+    {
+        $this->activeModulesPaths = null;
```

Comment:
Because `$this->activeModulesPaths` is an array, maybe we can do `$this->activeModulesPaths = [];` ?